numbers.go

```diff
@@ -51,3 +51,7 @@ func (n *StringOrNumber) UnmarshalJSON(data []byte) error {
 	}
 	return nil
 }
+
+func NilableInt(i int) *int {
```

Comment:
Could you pls call this just `func Int(i int) *int`? Could you pls also add a comment?